crates/pathfinder/src/rpc/v01.rs

```diff
@@ -2424,7 +2378,7 @@ mod tests {
             #[tokio::test]
             async fn declare_transaction() {
                 let storage = setup_storage();
-                let sequencer = Client::integration().unwrap();
+                let sequencer = Client::new(Chain::Integration).unwrap();
```

Comment:
This is actually a refactor I should have kept out of this PR as this is unrelated.